app/models/user.js

```diff
@@ -45,5 +45,16 @@ module.exports = (sequelize, DataTypes) => {
         }
         throw errors.databaseError(err);
       });
+
+  User.getUserByEmailAndPassword = email =>
```

Comment:
`getUserByEmailAndPassword` weird name for function, you can do more generic like `getUserBy = condition => ....`